ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -34,6 +37,11 @@ function prepare_next_steps() {
   fi
 }
 
+function update_config_map() {
```

Comment:
Let's put this in a separate step rather than duplicating this logic in every install step.